.github/workflows/build.yml

```diff
@@ -2,11 +2,7 @@ name: OpenVPNServ2
 
 on:
   push:
-    branches:
-     - master
```

Comment:
Is this what we want in master? Is this code not built in gerrit?

Reply:
I think only openvpn project itself is built in gerrit and the rest (including this) are not. I think GHA should be called for all branches, not only master.